src/pages/Groups/Group3/styles/Screen1.styled.ts

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
+
+export const MainContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
```

Comment:
![image](https://user-images.githubusercontent.com/63994179/137003746-9d4fb6f6-d463-428e-a09f-46aa99919041.png)
margin not needed in MainContainer (should not have space above the top nav bar), place the margin in the components within the container if it is needed instead (usual practice when styling rhapp screens)